Story/OnlineUIComponents/OnlinePlayerDisplay.cs

```diff
@@ -161,44 +164,59 @@ public override void Draw(float timeStacker)
                     color = Color.Lerp(color, new Color(1f, 1f, 1f), Mathf.InverseLerp(0f, 0.5f, Mathf.Lerp(this.lastBlink, this.blink, timeStacker)));
                 }
             }
-            var lighter_color = color * 1.7f;
-            this.username.color = lighter_color;
 
             if (this.message.text != "") // we've updated a username
             {
                 this.username.text = customization.nickname + ":";
                 resetUsernameCounter--;
                 this.username.color = color * 3f;
                 this.username.x = vector.x + this.message._textRect.x - 15f;
-
-
             }
 
-            if (resetUsernameCounter < 10) // snappier fadeaway
-            {
-                this.username.color = lighter_color;
-
-            }
-
-
             if (resetUsernameCounter < 0)
             {
-
                 this.message.text = "";
                 this.username.text = customization.nickname;
                 resetUsernameCounter = 200;
 
             }
 
-            this.arrowSprite.color = lighter_color;
-            this.slugIcon.color = lighter_color;
+            if (owner.PlayerInGate || owner.PlayerInShelter)
+            {
+                this.notifyPlayer++;
+
+                if (this.notifyPlayer % 60 == 0)
+                {
+                    notificationTimer = this.notifyPlayer;
+                    if (this.flippedColor)
+                    {
+                        if (this.message.text == "") this.username.color = Color.white;
+                        this.slugIcon.color = Color.white;
+                        this.arrowSprite.color = Color.white;
+                        this.flippedColor = false;
+                    }
+                    else if (!this.flippedColor)
```

Comment:
this if part is redundant, also consider flipping colour outside with foo = !foo

Reply:
see snippet at https://github.com/henpemaz/Rain-Meadow/pull/444#discussion_r1837347544